cinema/models.py

```diff
@@ -0,0 +1,13 @@
+from django.core.validators import MinValueValidator, MaxValueValidator
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=200)
+    description = models.TextField()
+    duration = models.IntegerField(
+        validators=[MinValueValidator(1), MaxValueValidator(300)]
+    )
+
+    class Meta:
+        verbose_name_plural = "movies"
```

Comment:
django provides the same by default
https://docs.djangoproject.com/en/4.2/ref/models/options/#verbose-name-plural

Reply:
ohhh, I didn't realize it... sorry... removed meta class

Reply:
np, you are learning, btw you didn't push it